js/web/lib/onnxjs/util.ts

```diff
@@ -476,7 +476,7 @@ export class LongUtil {
     if (Long.isLong(n)) {
       return n.toNumber();
     } else if (n instanceof flatbuffers.Long) {
-      return Long.fromValue({low: n.low, high: n.high, unsigned: true}).toNumber();
+      return Long.fromValue({low: n.low, high: n.high, unsigned: false}).toNumber();
```

Comment:
what is the change of this flag for?

Reply:
When ONNX model is converted to ORT model and there is no axis attribute in softmax, a tool will add an attribute 'axis' with a default value -1 into a model. floatbuffers.Long representation of int64 -1 is {low: -1, high: -1}. If 'unsigned' is true at Long type, a value will be 18446744073709552000, not -1.